src/main/resources/assets/emi_loot/lang/ru_ru.json

```diff
@@ -0,0 +1,348 @@
+  "emi_loot.function.set_stew_unknown": "неизвестного списка эффектов…",
+  "emi_loot.function.set_nbt": "NBT этих предметов задаётся с помощью пользовательских данных",
+  "emi_loot.function.set_loot_table": "Таблица добычи этого предмета установлена на %s",
+  "emi_loot.function.set_any_damage": "Предметы, поврежденные до любого значения прочности",
+  "emi_loot.function.ominous_banner": "Сброшенный капитаном рейда разбойников",
+  "emi_loot.condition.survives_explosion": "Должен пережить взрыв",
+  "emi_loot.condition.blockstate": "Нужен блок %s",
+  "emi_loot.condition.blockstate.block": "который является %s",
+  "emi_loot.condition.table_bonus": "Добыча увеличивается с %s",
+  "emi_loot.condition.invert": "НЕ: %s",
+  "emi_loot.condition.alternates": "%s",
+  "emi_loot.condition.alternates_2": "%1$s или %2$s",
+  "emi_loot.condition.alternates_3": "%s, ",
+  "emi_loot.condition.alternates_3a": "%s, или ",
+  "emi_loot.condition.killed_player": "Выпадает при убийстве игроком",
+  "emi_loot.condition.chance": "Выпадает %s%% от времени",
+  "emi_loot.condition.chance_looting": "%1$s%%  %2$s%% за каждый уровень Добычи",
+  "emi_loot.condition.damage_source": "Убит %s",
+  "emi_loot.condition.location": "Выпадает в определенных местах/мультиблоках",
+  "emi_loot.condition.entity_props": "Выпадает, когда %s",
+  "emi_loot.condition.match_tool": "Нужен предмет %s",
+  "emi_loot.condition.entity_scores": "выпадает, если сущность имеет определенный балл на табло",
+  "emi_loot.condition.reference": "Удаляется, если выполняется пользовательский не поддающийся разбору предикат %s",
+  "emi_loot.condition.time_check": "Выпадает, если время равно %s",
+  "emi_loot.condition.time_check_period": "Выпадает, если время в тиковом периоде %1$с равно %2$с",
+  "emi_loot.condition.value_check": "Выпадает, если %1$s равно %2$s",
+  "emi_loot.condition.weather_check": "Выпадает только в том случае, если %s",
+  "emi_loot.condition.weather_check.raining_true": "идёт дождь",
+  "emi_loot.condition.weather_check.raining_false": "не идёт дождь",
+  "emi_loot.condition.weather_check.thundering_true": "происходит гроза",
+  "emi_loot.condition.weather_check.thundering_false": "не происходит гроза",
+  "emi_loot.condition.direct_drop": "Выпадает напрямую, без использования таблицы добычи",
+  "emi_loot.condition.spawns_with": "Выпадает, если он был создан естественным образом вместе с мобом",
+  "emi_loot.condition.creeper": "Выпадает, когда этот моб взрывается заряженным крипером",
+  "emi_loot.condition.wither_kill": "Мобы, убитые Визером, сбрасывают это",
+  "emi_loot.number_provider.constant": "точно %s",
+  "emi_loot.number_provider.binomial": "результат %1$s тестов биномиального распределения периода $2%s (среднее значение %3$s)",
+  "emi_loot.number_provider.uniform": "между %1$s и %2$s (в среднем %3$s)",
+  "emi_loot.number_provider.score": "оценка объекта в %1$s, увеличенная в %3$s раз",
```

Comment:
What does "оценка" even has to do with anything here? This is used with the Statistics data, so it would be correct to write: "Статистика %1$s, увеличенная в %3$s раз"